app/screens/user_profile/user_profile.test.js

```diff
@@ -84,4 +87,70 @@ describe('user_profile', () => {
             />
         )).toEqual(true);
     });
+
+    test('should push EditProfile', async () => {
+        const props = {
+            ...baseProps,
+            navigator: {
+                push: jest.fn(),
+            },
+        };
+
+        const wrapper = shallow(
+            <UserProfile
+                {...props}
+                user={user}
+            />,
+            {context: {intl: {formatMessage: jest.fn()}}},
+        );
+
+        wrapper.instance().goToEditProfile();
+        setTimeout(() => {
+            expect(props.navigator.push).toHaveBeenCalledTimes(1);
+        }, 16);
```

Comment:
nit: Works the same with `0`, I recommend to change it, same case for similar instance.
```suggestion
        }, 0);
```

Reply:
@saturninoabril Thanks for the review.